deployment/deploy-kovan.ts

```diff
@@ -1,8 +1,111 @@
-async function main() {}
+import { getKovanSdk, getOptimismKovanSdk } from '@dethcrypto/eth-sdk-client'
+import { getRequiredEnv } from '@makerdao/hardhat-utils'
+import { expect } from 'chai'
+import * as dotenv from 'dotenv'
+import * as ethers from 'ethers'
+import { mapValues } from 'lodash'
+import { Dictionary } from 'ts-essentials'
+dotenv.config()
+
+import { deployOptimismWormholeBridge, OptimismBaseBridgeSdk, OptimismRollupSdk } from '../test/optimism'
+import { deployWormhole } from '../test/wormhole'
+import { performSanityChecks } from '../test/wormhole/checks'
+
+const bytes32 = ethers.utils.formatBytes32String
+
+async function main() {
+  const fee = 0 // 0 fees
+  const feeTTL = 60 * 60 * 24 * 8 // flush should happen more or less, 1 day after initWormhole, and should take 7 days to finalize
+  const ilk: string = bytes32('WH-KOVAN-TEST-1')
+  const masterDomain = bytes32('KOVAN-MASTER-1')
```

Comment:
Should we name this `KOVAN-MASTER` as there won't be more than one master domain per network (unlike for slave domains)?

Reply:
Also, is the MASTER/SLAVE necessary in domain ids? We could simply use `KOVAN` and `KOVAN-OPTIMISM-1`.

Reply:
> Should we name this KOVAN-MASTER as there won't be more than one master domain per network (unlike for slave domains)?

I dont' consider this deployment (even master) final one. I think we will add some small tweaks...

> Also, is the MASTER/SLAVE necessary in domain ids? We could simply use KOVAN and KOVAN-OPTIMISM-1.

I wanted to be explicit 🤷  I don't have a strong opinion on this.